web/locales/de.json

```diff
@@ -27,6 +27,7 @@
         "back": "zurück",
         "source": "Quelle",
         "fossilfuels": "fossile Brennstoffe",
+        "carbonintensity": "CO2 Konzentration",
```

Comment:
Of course i can fix it, but I don't like to fix language issues of a language i don't understand. To easy to make it wrongly. 

I think this should be fixed in a separate PR so we not mix different type of changes.